srcpkgs/factor/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'factor'
+pkgname=factor
+version=0.97
+revision=1
+only_for_archs="x86_64"
+build_style=gnu-makefile
+hostmakedepends="unzip pkg-config"
+makedepends="gtkglext-devel"
+nostrip_files="a.elf"
+short_desc="Factor: a concatenative programming language, similar to Forth"
+maintainer="B. Wilson <x@wilsonb.com>"
+license="BSD"
```

Comment:
Please specify which BSD license using SPDX https://spdx.org/licenses/

Reply:
Excellent. Thank you. Fixed.